### ISSUE TITLE
[Tooltip] accessibility improvements

### ISSUE THREAD

**Issue by owestinTelus** [telus/tds-core#434]
## Description

The accessible toggle tooltip can be improved.

- change role="polite" to assertive
- update the code source so that the tooltip content is after the toogle button

## Reproduction Steps
- [ ] Activate text to speech enging (screen reader)
- [ ] Select the toggle tooltip with a keyboard
- [ ] Activate the button with spacebar or enter
- [ ] Toggle state of button is announced first, followed by the tooltip
- [ ] Use screen reader navigation left to read content manually (should be after button > navigate right)

## Reproduction Percentage (if applicable)
- NA

## Version Number of TDS
- [ ] v 0.29

*note attach screenshots whenever possible*
